mscore/scoreview.cpp

```diff
@@ -2916,9 +2916,8 @@ void ScoreView::adjustCanvasPosition(const Element* el, bool playBack, int staff
                   showRect.setHeight(r.height());
                   }
             }
-      if (mscore->state() & ScoreState::STATE_NOTE_ENTRY) {
+      if (_score->inputState().noteEntryMode())
```

Comment:
This looks like it amounts to the same thing, but it really makes all the difference. The reason is because Seq::start() is called in the middle of ScoreView::changeState(). It comes after ScoreView::endNoteEntry(), but before ScoreView::state is updated, and before mscore->changeState() is called.